modules/plottinggl/glsl/pcp_lines.frag

```diff
@@ -4,11 +4,14 @@ in float lfalloffAlpha;
 
 uniform bool additiveBlend;
 uniform float alpha;
+uniform float filteredAlpha;
```

Comment:
Why not use filteredColor.w?

Reply:
Yeah that would work as well. But it's a little bit cumbersome to change the alpha of filteredColor in the GUI compared to a separate alpha-slider. Perhaps a solution would to keep the slider but synchronize it with filteredColor.w and then use that in the shader?